src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -400,11 +400,21 @@
                     
                     <te:SectionControl.Content>
                         <Grid>
+                            <Grid.Resources>
+                                <!-- We have to do contortions to get to the PullRequestDetailViewModel which contains the OpenFile and DiffFile
+                                     commands: the control which hosts the context menu must have the PullRequestDetailViewModel as its tag,
+                                     which is then set as the tag on the ContextMenu which is finally reacable from the MenuItem. -->
+                                <ContextMenu x:Key="FileContextMenu" Tag="{Binding PlacementTarget.Tag, RelativeSource={RelativeSource Mode=Self}}">
```

Comment:
Yeah, I considered this but adding a separate command to each changed file (together with a reference to the parent view model) just to avoid this seemed wasteful to me.